Explicit SVG support in godot (#7590)
godotengine/godot-docs: tutorials/assets_pipeline/importing_images.rst

- Truevision Targa (``.tga``)
- SVG (``.svg``, ``.svgz``)
  - SVGs are rasterized using `ThorVG <https://www.thorvg.org/>`__

>   when importing them. ThorVG only support the `tiny SVG specification <https://www.w3.org/TR/SVGTiny12/>`__ 

Reviewer: ```suggestion
  when importing them. ThorVG only supports the `SVG 1.2 Tiny specification <https://www.w3.org/TR/SVGTiny12/>`__ 
```

>   to keep it lightweight ( yet unsupported tiny features are : Animation, Fonts & Text, Interactivity and Multimedia); 

Reviewer: ```suggestion
  to keep it lightweight. Some features of the Tiny specification are **not** supported : animation, fonts and text, interactivity and multimedia;
```

>   so complex vectors may not render correctly. 

Reviewer: ```suggestion
  As a result, complex vectors may not render correctly. 
```
